extensions/spotify-player/src/api/oauth.ts

```diff
@@ -2,81 +2,81 @@ import { OAuth } from "@raycast/api";
 import fetch from "node-fetch";
 import { URLSearchParams } from "url";
 
-const clientId = "7bbb789c01ff44ed842907b7a80c404f";
-const scope = "user-library-modify user-modify-playback-state";
-//
-// user-read-currently-playing
-const oauthClient = new OAuth.PKCEClient({
+const clientId = "b997e8ba5e5f473eb3f846666d85f064";
+const scope = "user-library-modify user-modify-playback-state user-read-currently-playing user-library-read";
```

Comment:
**FYI:** Because the Oauth app has new scopes, you would need to log out existing users. Unsure if we have that capability in the API yet. A work-around could be to use a new `providerId`, which is used to retrieve the credentials.